### PR TITLE
website improvement

### PR DESCRIPTION
**Pull request by JaninaBodi**
Improved the website to make it look more like Mar's design, some things are still not 100% in line so I'll continue working on that.

### CODE REVIEW

**Review by marcoow — COMMENTED** [review]
Looks good – just one thing I recognized: the chameleon next to the form is a bit distorted on medium size viewports:

Bildschirmfoto 2020-09-28 um 13 45 22

File: css/app.scss [hunk 1]
```diff
@@ -544,14 +615,71 @@ label {
...
+        display: none;
+      }
+
+      .mobile {
+        display: none;
+      }
+
+      .tablet {
+        display: block;
+        display: block;
```
Comment (marcoow) [inline css/app.scss:677]: this seems duplicated

File: css/app.scss [hunk 2]
```diff
@@ -131,7 +136,16 @@ a {
 }
 
 .header__landing-page__date {
-  font-size: 14px;
+  margin-bottom: var(--space-m);
+}
+
+
+#content-mobile {
+}
```
Comment (marcoow) [inline css/app.scss:144]: this shouldn't be needed?

File: css/app.scss [hunk 3]
```diff
@@ -659,14 +823,28 @@ label {
...
+  }
+
+  .pre-heading {
+    margin-top: var(--space-xl);
   }
 
   .video-2019 {
     padding-left: var(--space-xl);
     padding-right: var(--space-xl);
+    height: 1146px;
```
Comment (marcoow) [inline css/app.scss:847]: we shouldn't be setting `height` on `div`s but let the content of the `div` determine its `height`

File: css/app.scss [hunk 4]
```diff
@@ -477,14 +514,30 @@ label {
...
     margin-left: var(--space-m);
     margin-right: var(--space-m);
+    margin-top: var(--space-l);
   }
 
   .video-2019 {
     padding-left: var(--space-m);
     padding-right: var(--space-m);
+    background-image: url(/images/video-2019-background-tablet.svg);
+    height: 727px;
```
Comment (marcoow) [inline css/app.scss:535]: see below – we shouldn't be setting `height` on `div`s

**Review by marcoow — COMMENTED** [review]
👍 some minor things left to fix

**Review by marcoow — CHANGES_REQUESTED** [review]
the chameleon is too big now on tablets:

deploy-preview-214--silly-leavitt-2e0668 netlify app_

**Comment by JaninaBodi** [comment]
@marcoow Oh, that's weird, I didn't change anything and when I look at it, it has the correct size, so I don't really know what to change?

**Comment by marcoow** [comment]
@JaninaBodi change the width of the window a bit – it's off for widths between mobile and desktop only

**Comment by JaninaBodi** [comment]
@marcoow I think I found the problem, is it the right size now? I mean it is when I zoom in or out on the window on my laptop

**Review by marcoow — APPROVED** [review]

**Pull request merged**